lib/hotel_front_desk.rb

```diff
@@ -0,0 +1,283 @@
+require_relative 'lib_requirements.rb'
+require_relative 'room'
+require_relative 'reservation'
+require_relative 'date_range'
+
+class Hotel_front_desk
```

Comment:
By convention, class names use camel case so this would be named HotelFrontDesk (unlike the file name).

Reply:
fixed